### PR TITLE
Use Artemis base image, upgrade to 2.2.0 and use secrets for the conf…

### PR DESCRIPTION
**Pull request by dejanb**
…iguration

### CODE REVIEW

File: bom/pom.xml [hunk 1]
```diff
@@ -30,6 +30,7 @@
 
   <properties>
     <artemis.version>2.1.0</artemis.version>
+    <artemis.image.name>enmasseproject/activemq-artemis:2.2.0-1</artemis.image.name>
```
Comment (sophokles73) [inline bom/pom.xml]: shouldn't we remove `artemis.version` in this case? Or at least use the same version (2.2.0)?

File: example/src/main/deploy/docker/swarm_deploy.sh [hunk 2]
```diff
@@ -36,7 +36,27 @@ echo ... done
 
 echo
 echo Deploying Artemis broker ...
-docker service create $CREATE_OPTIONS --name hono-artemis eclipsehono/hono-artemis:${project.version}
+docker secret create -l $NS broker.xml $CONFIG/hono-artemis-jar/etc/broker.xml
+docker secret create -l $NS bootstrap.xml $CONFIG/hono-artemis-jar/etc/bootstrap.xml
```
Comment (sophokles73) [inline example/src/main/deploy/docker/swarm_deploy.sh]: maybe we could prefix these two secret names with `artemis-` as well in order to indicate their relation to Artemis?

**Comment by dejanb** [comment]
@sophokles73 thanks for the feedback. I made appropriate changes. Regarding secret names, there's still some work to be done, to improve Artemis config in this environment. I'll see to do this for the next image upgrade.

**Comment by sophokles73** [comment]
@dejanb 
any reason you are not merging this PR?

**Pull request merged**